README.md

```diff
@@ -77,3 +108,17 @@ Don’t forget to add the *Spotipy* tag, and any other relevant tags as well, be
 If you have suggestions, bugs or other issues specific to this library,
 file them [here](https://github.com/plamere/spotipy/issues).
 Or just send a pull request.
+
+## Contributing
+
+If you are a developer with Python experience, and you would like to contribute to Spotipy, please be sure to follow the guidelines listed on documentation page
+
+> #### [Visit the guideline](https://spotipy.readthedocs.io/en/2.22.1/#contribute)
+
+## License
```

Comment:
I'm not sure if this section is needed, as GitHub has very recently added tabs at the top to link to the licence among others:

<img width="541" alt="image" src="https://github.com/spotipy-dev/spotipy/assets/1324225/aea289e3-fadf-4104-93dc-def2ab0ac4a6">